corehq/apps/reports/templatetags/xform_tags.py

```diff
@@ -37,8 +37,8 @@ def render_form_xml(form):
     if isinstance(xml, unicode):
         xml.encode('utf-8', errors='replace')
     formatted_xml = indent_xml(xml) if xml else ''
-    return '<pre class="prettyprint linenums"><code class="no-border language-xml">%s</code></pre>' \
-           % escape(formatted_xml)
+    return format_html('<pre class="prettyprint linenums"><code class="no-border language-xml">{}</code></pre>',
+                       formatted_xml)
```

Comment:
I'm not seeing line numbers. Should I be? 

And to render the XML I dropped the `escape()` call. I didn't expect that either.